_posts/2023-01-25-snowflake-snowpro-core-cert.md

```diff
@@ -0,0 +1,244 @@
+---
+layout: postv2
+hidden: true
+inject_optimize_tag: true
+canonical_url: https://mechanicalrock.github.io/2023/01/25/snowflake-snowpro-core-cert.html
+font: serif
+title: "Snowflake SnowPro Core Certification notes"
+description: ""
+date: 2023-01-25
+highlight: monokai
+image: /img/skeleton-loaders/banner.jpg
+author: Zainab Maleki
+tags: [snowflake, snowpro, core, certification, cheatsheet, exam, notes]
+---
+
+# Snowflake SnowPro Core Certification cheat sheet
+
+After working with Snowflake for almost four years I have finally bowed down to studying for the Snowflake certification and passed my exam this morning. Before I started going through the practice tests I was thinking with four years of experience passing the exam should be fairly easy. However soon after going through my first online practice test, I realized the practical experience of using Snowflake plus intuition and some luck would probably only get you less than 50% of the score and you will need a lot of studying for things that you don't normally use or you would google whenever you come across those requirements. Unfortunately googling is not an option during the test, hence this cheat sheet might come handy if you are studying for this cert.
```

Comment:
- `with four years of experience passing, the exam should be fairly easy`. Add a comma between `passing` and `the exam`.
-